features/font-variant-emoji.yml

```
@@ -0,0 +1,6 @@
+name: Emoji font variants
+description: The `font-variant-emoji` CSS property sets the default presentation style for emoji characters.
```

I think this is very hard to describe and the word "style" is a bit overloaded when it comes to CSS features. What do you think of either of these variations (pun intended)?

Maximalist rewrite:

```suggestion
description: The `font-variant-emoji` CSS property overrides how the browser picks which emoji character to use when presentation variations exist.
```

Minimalist rewrite:

```suggestion
description: The `font-variant-emoji` CSS property sets the default presentation for emoji characters.
```

---

Agreed, this was one of the harder ones to write. I went with the minimalist rewrite, as the maximalist wasn't quite accurate- it isn't which emoji character is used, but rather whether or not emoji is selected or not.